### PR TITLE
Show a list of tokens and their annotations

### PR DESCRIPTION
**Pull request by paltman**

### CODE REVIEW

**Comment by paltman** [comment]
[As a user, I can view a widget that displays morphosyntactic annotations for a particular passage and select a token](https://trello.com/c/O2Z7aBxj/101-as-a-user-i-can-view-a-widget-that-displays-morphosyntactic-annotations-for-a-particular-passage-and-select-a-token)

File: src/widgets/TokenAnnotationWidget.vue [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            passageTextParts(reference: "${this.urn.absolute}") {
+              edges {
+                node {
+                  id
+                  ref
+                  tokens {
+                    edges {
+                      node {
+                        veRef
+                        value
```
Comment (jacobwegner) [inline src/widgets/TokenAnnotationWidget.vue:55]: `wordRef` would probably be better here (will omit punctuation)

**Comment by jacobwegner** [comment]
Easier preview URL (because the default site URL loads the Venetus A):

https://deploy-preview-8--explorehomer-dev.netlify.app/reader?urn=urn%3Acts%3AgreekLit%3Atlg0012.tlg001.perseus-grc2%3A1.1-1.7

File: src/widgets/TokenAnnotationWidget.vue [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  import gql from 'graphql-tag';
+  import { URN } from '@scaife-viewer/scaife-widgets';
+  import { MODULE_NS, CLEAR_TOKEN } from '@/reader/constants';
+
+  export default {
+    scaifeConfig: {
+      displayName: 'Token Annotations',
+    },
+    methods: {
+      prettyPrint(token) {
```
Comment (jacobwegner) [inline src/widgets/TokenAnnotationWidget.vue:24]: @paltman I know that the "guts" of this widget came from my spike; we can probably drop `prettyPrint` now.

File: src/widgets/TokenAnnotationWidget.vue [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+          return [...new Set(tokens)];
+        }
+        return [];
+      },
+    },
+  };
+</script>
+
+<style lang="scss" scoped>
+  @import '../styles/variables';
```
Comment (jacobwegner) [inline src/widgets/TokenAnnotationWidget.vue:107]: We should probably match indentation to the other widgets:

File: src/widgets/TokenAnnotationWidget.vue (hunk 1 above)
Comment (paltman) [reply src/widgets/TokenAnnotationWidget.vue:55]: I get a `400` when trying to use `wordRef`

Comment (paltman) [reply src/widgets/TokenAnnotationWidget.vue:55]: Screen Shot 2020-05-14 at 9 44 11 PM

Comment (paltman) [reply src/widgets/TokenAnnotationWidget.vue:55]: I think `wordValue` is what you meant.  that seems to do it.

**Comment by paltman** [comment]
@jacobwegner Feedback addressed. Thanks!

**Pull request merged**